test/langtools/tools/javac/valhalla/lworld-values/InferredValueParameterizationTest.java

```diff
@@ -1,13 +1,40 @@
 /*
- * @test /nodynamiccopyright/
- * @bug 8210346
- * @summary inferred value typed `type arguments' are allowed by Javac even without -XDallowGenericsOverValues
- * @compile/fail/ref=InferredValueParameterizationTest.out -XDrawDiagnostics -XDdev InferredValueParameterizationTest.java
+ * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The golden file seems to still be there  - even though this is now a positive test

Reply:
Thanks, will remove it!